### ISSUE TITLE
Not saving session?

### ISSUE THREAD

**Issue by silvestreh** [thomaslindstrom/atom-project-environment#1]
I'm not sure if it's just an isolated problem, but when I use this package Atom won't remember which files I had open the next time I open my project that takes advantage of this package's features.

It does populate the treeview with the correct paths. Directories appear expanded by default and it doesn't re-open the files I've been working on during the last session.

By *expanded* I mean this:

![captura de pantalla 2015-12-23 a las 12 35 05](https://cloud.githubusercontent.com/assets/1133339/11979385/ad40a86c-a971-11e5-8258-7851088c85ac.png)

**Comment by thomaslindstrom** [comment]
Yeah, I have been noticing this too - but it also appears to be saving the state sporadically. Not sure if this could be done programmatically (trigger a save), but it's definitely sub-par right now. 

Any help is appreciated, but I'll definitely try and take a look at this pretty soon!

**Comment by thomaslindstrom** [comment]
https://atom.io/docs/latest/behind-atom-serialization-in-atom
